nifi-assembly/pom.xml

```diff
@@ -363,147 +363,6 @@ language governing permissions and limitations under the License. -->
         </dependency>
     </dependencies>
 
-    <properties>
```

Comment:
Yup I did mean the toolkit assembly, sorry for the confusion. 

Ah that is cool, sounds good.